lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -96,6 +105,8 @@ CHECK_EXISTING_IDENTITY:
   events:
     next:
```

Comment:
Maintaining the old `next` event out of a fear of a race condition. If the check-existing-identity lambda is deployed before the process-journey-event lambda we could get some unknown event errors. Maybe this isn't something to worry about too much as the timing might be very small.

Reply:
Yeah, I think it'd be a small enough window (<1s) that we could get away without it, but if we can easily handle it then we may as well.

TBD exactly how the blue/green deployments will work for core-back, but there's a chance it might avoid this anyway in future (by pinning lambda versions to each step function version).